### ISSUE TITLE
Bug (may be)

### ISSUE THREAD

**Issue by MikhailLipanin** [yegor256/bibcop#3]
I have a citation with this title:
```
title = {{Transformation of C Programming Language Memory Model into Object-Oriented Representation of {EO} Language}},
```
And `bibcop` gets this warning:
```
Package bibcop Warning: All major words in the 'title' must be capitalized, while 'into' (no.8) is not, in the 'legalov2022' bibitem.
```

but my title already looks good, as it is described in warning.

@yegor256 FYI

**Comment by yegor256** [comment]
@MikhailLipanin should be `Transformation of C Programming Language Memory Model Into Object-Oriented Representation of {EO} Language}`. Pay attention, the word `Into` is capitalized.
